how-to/business-logic/how-to-determine-requested-item-id.md

```diff
@@ -0,0 +1,55 @@
+---
+title: How to Determine the Id of the Requested Item
+description: An example to determine the Id of the requested item
+type: how-to
+page_title: How To Determine the Id of the Requested Item in the {{site.bs-bl}} Layer?
+slug: bl-id-requested-item
+position:
+tags: backendservices, business-logic, cloud-code
+teampulseid: 
+ticketid: 
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Business Logic</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Cloud Code for Data, Cloud Functions</td>
+  </tr>
+</table>
+
+## Description
+
+**Applicable only to {{site.bl-data}}**. Frequently you may need to obtain the `Id` of a requested data item for your data requests per Id. This is only applicable to Read, Update and Delete requests and their respective events in the {{site.bl-data}}. 

```

Comment:
Suggestion: Use "often" instead of "frequently"

Consider rephrasing the "requested data item for your data requests" to avoid the redundancy.

This is only applicable -> This is applicable only to...

Have you considered code formating for the request names?